gapii/cc/spy_base.h

```diff
@@ -112,6 +112,9 @@ class SpyBase {
   void set_record_timestamps(bool record) { mRecordTimestamps = record; }
   bool should_record_timestamps() const { return mRecordTimestamps; }
 
+  // Checks if the current trace should be ended
+  virtual bool checkEndTrace() { return false; }
```

Comment:
Since the **only** person who overrides this is (eventually) spy, this should probably be pure virtual.

Reply:
Leaving it for now, because pure virtual Spy function triggers compilation error in GlesSpy constructor.